src/uwtools/tests/drivers/test_cdeps.py

```diff
@@ -35,6 +36,14 @@ def driverobj(tmp_path):
 # Tests
 
 
+@mark.parametrize(
+    "method",
+    ["taskname", "_validate"],
+)
+def test_CDEPS(method):
+    assert getattr(CDEPS, method) is getattr(AssetsCycleBased, method)
```

Comment:
The other `Assets` drivers provide this test, so I added a copy to the CDEPS test module.